src/geode/mesh/builder/solid_mesh_builder.cpp

```diff
@@ -280,6 +280,76 @@ namespace
             builder.create_polyhedron( vertices, facets );
         }
     }
+
+    template < geode::index_t dimension >
+    void update_edge_and_facet( const geode::SolidMesh< dimension >& solid,
+        geode::SolidMeshBuilder< dimension >& builder,
+        const geode::PolyhedronVertex& polyhedron_vertex,
+        geode::index_t new_vertex_id )
+    {
+        for( const auto f : geode::LRange{ solid.nb_polyhedron_facets(
+                 polyhedron_vertex.polyhedron_id ) } )
+        {
+            const geode::PolyhedronFacet id{ polyhedron_vertex.polyhedron_id,
+                f };
+            const auto nb_facet_vertices =
+                solid.nb_polyhedron_facet_vertices( id );
+            absl::FixedArray< geode::PolyhedronVertex > facet_vertices(
+                nb_facet_vertices );
+            for( const auto v : geode::LRange{ nb_facet_vertices } )
```

Comment:
Is there a function that return the result of this for loop? Seems an interesting feature.

Reply:
not yet. I already made the `polyhedron_facet_vertex_id` public. It will allow us to think about other APIs I am sure.